src/composables/staking/useStaking.ts

```diff
@@ -1,13 +1,3 @@
-import { inject } from 'vue';
-
-import { StakingProviderSymbol } from '@/providers/local/staking/staking.provider';
-
 export default function useStaking() {
-  const providedData = inject(StakingProviderSymbol);
-  if (!providedData) {
-    throw new Error(
-      `useStaking was called from a component that was not a child of <StakingProvider />`
-    );
-  }
-  return providedData;
+  return {};
```

Comment:
Cannot we delete this file?

Reply:
I added something useful to this file in the Gnosis PR, but yeah I should probably delete here. Then we can move that reference in the Gnosis PR somewhere else.

Reply:
Removed.